samples/batch/transcription-enabled-storage/Connector/StorageConnector.cs

```diff
@@ -142,8 +159,18 @@ public async Task MoveFileAsync(string inputContainerName,  string inputFileName
             var outputContainerClient = BlobServiceClient.GetBlobContainerClient(outputContainerName);
             var outputBlockBlobClient = outputContainerClient.GetBlobClient(outputFileName);
 
+            if (await outputBlockBlobClient.ExistsAsync().ConfigureAwait(false))
```

Comment:
What happens if the file is only partially written?

Reply:
right.. probably I should override it here